source/docs/software/support/support-resources.rst

```diff
@@ -27,7 +26,7 @@ Stuck? Have a question not answered by the documentation? Official Support is pr
 NI Phone Support
 ----------------
 
-Have a LabVIEW, roboRIO, or Driver Station question? NI provides phone support for FRC teams during the build season at `+1 (866) 511-6285 <tel:+1-866-511-6285>`__ 1:00-7:00 PM CST Monday - Friday.
+Have a LabVIEW, roboRIO, or Driver Station question? NI provides phone support for FRC teams during the build season at `+1 (866) 511-6285 <tel:1-866-511-6285>`__ 1:00-7:00 PM CST Monday - Friday.
```

Comment:
This doesn't seem right. You need the `+` when calling from outside the US or else it will not go through.

Reply:
Errrrgh. The + breaks linkcheck for some reason.

Reply:
Opened an issue here https://github.com/sphinx-doc/sphinx/issues/8321

Reply:
Okay, removing the dashes seems to have also fixed it.

Reply:
Looks like linkcheckdiff is not reporting the failure on the phone link 🤔